toxcore/tox_private.h

```diff
@@ -139,6 +139,13 @@ typedef enum Tox_Err_Dht_Get_Nodes {
 bool tox_dht_get_nodes(const Tox *tox, const uint8_t *public_key, const char *ip, uint16_t port,
                        const uint8_t *target_public_key, Tox_Err_Dht_Get_Nodes *error);
 
+/**
+ * This function returns the ratio of close dht nodes that are known to support announce/store.
```

Comment:
What will the client do with this information?

Reply:
This info is purely diagnostic, to get the ratio of connected DHT nodes that are >= `0.2.18`. Thats why it is in `tox_private.h`.
I am thinking of redesigning this, so you can get the raw numbers (total- & new-nodes).

Reply:
Raw numbers is better, I think. It avoids floats and gives additional information.